labelme/labelDialog.py

```diff
@@ -90,19 +90,15 @@ def postProcess(self):
             self.edit.setText(self.edit.text().trimmed())
 
     def popUp(self, text=None, move=True):
-        if isinstance(text, str):
-            self.edit.setText(text)
-            self.edit.setSelection(0, len(text))
-            i = self.labelList.findItems(text, Qt.MatchFixedString)
-            if len(i) == 1:
-                self.labelList.setCurrentItem(i[0])
+        if text is None:
```

Comment:
The reason I had put the complete code block inside the if was for the case of labeling many instances of the same label in 1 image. In that case it is convenient to not have to enter the label name again, but instead still having it from the previous label action.

If you prefer to not keep the previous label in the editbox, I would also suggest to add the following block below:

```py
if items:
    assert len(items) == 1
    self.labelList.setCurrentItem(items[0])
else:
    self.labelList.clearSelection()
```

Reply:
I think keeping the previous label is a good idea.
However, how can it be achieved?
Where do you keep the previous label?

Reply:
Ah, now I understand. And I found it is necessary to ensure `text=None`.